README.md

```diff
@@ -45,6 +45,10 @@ generate this data from a wikidata dump.
 
 ## Release notes
 
+### 2.3.0 (2015-06-26)
+* Replace use of Wikibase\TermIndex::getMatchingIDs with Wikibase\TermIndex::getTopMatchingTerms.
+* EntitySelector no longer passes "type" parameter to wbsgetsuggestions which avoids an "Unrecognized parameter" warning.
```

Comment:
Oh. Starts making sense again. Thanks a lot for the explanation, @JanZerebecki. Very much appreciated.